packages/components/src/components/hds/advanced-table/index.ts

```diff
@@ -246,11 +246,11 @@ export default class HdsAdvancedTable extends Component<HdsAdvancedTableSignatur
 
     // if there is no custom column widths, each column is the same width and they take up the available space (except the select checkbox)
     if (!this.columnWidths) {
-      return `${isSelectable ? 'auto ' : ''}repeat(${columns.length}, 1fr)`;
+      return `${isSelectable ? 'min-content ' : ''}repeat(${columns.length}, 1fr)`;
     }
 
-    // if there is a select checkbox, the first column is 'auto' width to hug the checkbox content
-    let style = isSelectable ? 'auto' : '';
+    // if there is a select checkbox, the first column is 'min-content' width to hug the checkbox content
+    let style = isSelectable ? 'min-content ' : '';
```

Comment:
Technically in Figma this is element is 48x48px, which referencing the showcase the dimensions are 49x48.

Reply:
@jorytindall does that include the border?

Reply:
@shleewhite good point, Figma doesn't visually represent the DOM in the same way, so no, that doesn't include the border; technically the border is "inside" of the element, which is a conceptual difference.

Reply:
+1 on the border not being an issue due to how Figma treats the border (plus, in a browser the border is always outside)